File: Xicom-backend/src/main/java/com/example/xicombackend/entity/Abonnee.java
package com.example.xicombackend.entity;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@NoArgsConstructor
@Table(name = "abonnee")
public class Abonnee {
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    private Long id;

    private String name;
    private String email;
}

File: xicom-angular/src/app/models/abonnee.ts


export class Abonnee {
    private id : any
    private email: string;
    private name: string;


  
    constructor(
      id: any,
      email: string,
      name: string,



    ) {
      this.id = id;
      this.email = email;
      this.name = name;

    }
  
    public get Id(): any {
      return this.id;
    }

  
    public get Email(): string {
      return this.email;
    }
  
    public set Email(email: string) {
      this.email = email;
    }
  
  
    public get Name(): string {
      return this.name;
    }
  
    public set Name(name: string) {
      this.name = name;
    }


    
  }